### PR TITLE
update menu and registration

### PR DESCRIPTION
**Pull request by projectmushroom**
Add a required asterisks to registration page, replace menu icons with text

**Before:**
screen shot 2017-10-20 at 14 08 08

**After:**
screen shot 2017-10-20 at 14 06 12

### CODE REVIEW

File: iogt/styles/iogt/modules/_m-profiles.scss [hunk 1]
```diff
@@ -68,3 +68,10 @@ label.profiles__terms {
   color: red;
   font-size: $font-small;
 }
+.required:after {
```
Comment (nathanbegbie) [inline iogt/styles/iogt/modules/_m-profiles.scss:71]: Is this going to work with feature phones?

**Comment by nathanbegbie** [comment]
@projectmushroom Could you post a screenshot of what the low-end version looks like please? 
We could add the `*` within the html template instead? I'm just not sure whether it will render reliably on a feature phone.

File: iogt/styles/iogt/modules/_m-profiles.scss (hunk 1 above)
Comment (projectmushroom) [reply iogt/styles/iogt/modules/_m-profiles.scss:71]: yes, been using pseudo elements on most of our molo projects, no issues yet, its widely supported on older browsers and most importantly all versions of opera mini, https://caniuse.com/#feat=css-gencontent

screen shot 2017-10-20 at 14 28 35

Comment (nathanbegbie) [reply iogt/styles/iogt/modules/_m-profiles.scss:71]: Cool - thanks for the explanation :)

**Comment by nathanbegbie** [comment]
:+1:

**Pull request merged**